implementations/rust/ockam/ockam_command/tests/bats/portals_orchestrator.bats

```diff
@@ -79,6 +79,7 @@ teardown() {
   run_success "$OCKAM" node create green --project-path "$PROJECT_JSON_PATH" --identity green
   run_success "$OCKAM" node create blue --project-path "$PROJECT_JSON_PATH" --identity blue
 
+  -- TO_REVIEW --
```

Comment:
What happens in this kind of scenario, where the admin enrolls a new identity as a member of the project? Should we also include the `--allowed-relay` argument here?